core/scripts/generate.cjs

```diff
@@ -47,15 +47,23 @@ const generate = async () => {
   await generateSchema({
     input: getEndpoint(),
     headers: { Authorization: `Bearer ${getToken()}` },
-    output: join(__dirname, '../schema.graphql'),
+    output: join(__dirname, '../bigcommerce.graphql'),
     tsconfig: undefined,
   });
 
   await generateOutput({
     disablePreprocessing: false,
-    output: join(__dirname, '../graphql-env.d.ts'),
```

Comment:
Why are we removing this? Shouldn't this point to `bigcommerce-graphql.d.ts` now?

Reply:
While in Multi-schema mode, since `generateOutput` produces an introspection file for each GraphQL schema in the `tsconfig` plugin `schemas` array, we can't pass it a single file name - instead, `generateOutput` will create a file for each `tadaOutputLocation` key in each schema config object 🙂 

If we try to manually set this to:
```ts
await generateOutput({
  disablePreprocessing: false,
  output: join(__dirname, '../bigcommerce-graphql.d.ts'),
  tsconfig: undefined,
});
```

We get the following:
<img width="659" alt="Screenshot 2024-06-07 at 2 24 48 PM" src="https://github.com/bigcommerce/catalyst/assets/28374851/4bcf26f6-b68a-4491-9512-93e66c42b909">